### ISSUE TITLE
Does it make sense that the validation loss is lower than training loss?

### ISSUE THREAD

**Issue by paipai880429** [fchollet/keras#472]
I have used Keras for Kaggle-Facial Detection. When I plot the training and validation loss over epochs (depend on the callbacks.History), I find out that the validation loss is even lower than training loss. Does this make sense? As far as I understand, it should be a bit higher than training loss. Here is the plot.

![image](https://cloud.githubusercontent.com/assets/3724085/9031941/1a6f1414-39ec-11e5-98a2-6cf8aaac31ba.png)

**Comment by paipai880429** [comment]
I think this is a sign of overfitting. After I add several dropout layers after convolutional layers, I get a more reasonable figure. 
![image](https://cloud.githubusercontent.com/assets/3724085/9032537/560547d6-39f2-11e5-8e0c-31f625c9beea.png)

**Comment by rohun-tripathi** [comment]
For future users - https://keras.io/getting-started/faq/#why-is-the-training-loss-much-higher-than-the-testing-loss
